.circleci/config.yml

```diff
@@ -416,8 +418,9 @@ jobs:
       - run:
           name: Start daemon services
           command: |
-            cd priv/cabbage
-            make start_daemon_services-2 || (START_RESULT=$?; docker-compose logs; exit $START_RESULT;)
+            make init_test
+            cp ./localchain_contract_addresses.env ./priv/cabbage/apps/itest/localchain_contract_addresses.env
+            docker-compose -f docker-compose.yml -f docker-compose.feefeed.yml -f docker-compose.specs.yml up -d || (START_RESULT=$?; docker-compose logs; exit $START_RESULT;)
```

Comment:
maybe it makes sense to add this into makefile?
do we have a README on how to run cabbage tests now (for local development) so that engineers can use that?

Reply:
I moved them to `Makefile`. Also, I updated the README